setup.py

```diff
@@ -17,17 +17,15 @@
     author_email="tech@opensafely.org",
     python_requires=">=3.7",
     install_requires=[
-        "lz4",
```

Comment:
I seem to remember this is needed for compressed feather files. Once we confirm the situation we should add a comment here to explain.

Reply:
Ah, ok, let me dig into it.

There were no tests failing when removing it, so...

Reply:
https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather

Note that support for that was added in pandas 1.1.0, we have 1.0.1, so we had the dependency, but it wasn't actually being used!

I will restore it, I'm doing an upgrade PR next.

Reply:
I _think_ that, helpfully, it might just write uncompressed files if LZ4 isn't available:
https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather

Reply:
Ah, no https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_feather.html the kwargs was unsupported, but the underlying defaults were valid.

Anyway, will restore, and am writing functional tests for these formats to make sure we can write all the formats as expected.

Reply:
restored